test/e2e/main_test.go

```diff
@@ -112,6 +116,15 @@ func TestMain(m *testing.M) {
 		Setup(func(t resource.Context) error {
 			return SetupApps(t, i, apps)
 		}).
+		Setup(
+			func(t resource.Context) (err error) {
```

Comment:
```suggestion
			Setup(func(t resource.Context) (err error) {
			      prom, err = prometheus.New(t, prometheus.Config{})
			      return
			}
```

The format should be consistent with other `Setup`